nysescrape.py

```diff
@@ -0,0 +1,58 @@
+############################################################################
+
+
+from selenium.webdriver.chrome.options import Options
+from selenium import webdriver
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.ui import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+from bs4 import BeautifulSoup
+
+
+def getnyseSoup(ticker):
+    url = "https://au.finance.yahoo.com/quote/" + ticker + "?p=" + ticker + "&.tsrc=fin-srch"
```

Comment:
Try changing to this url https://au.finance.yahoo.com/quote/TSLA, verify that it still works